### PR TITLE
Add ETH send PayID demo

### PR DESCRIPTION
**Pull request by Stormtv**
## High Level Overview of Change
Adds an example of sending ETH using PayID

### Context of Change
Was asked to move those into this library for showcasing this going forward.

### Type of Change
- [X] New feature (non-breaking change which adds functionality)

## Before / After
Added a new eth demo

## Test Plan
Replace the Address and Private Key with proper values in the ETH demo (Reach out to me on slack if u need a keypair)

npm install then run node ./src/index-send-eth-to-payid

### CODE REVIEW

**Review by amiecorso — CHANGES_REQUESTED** [review]
This is awesome Tyler! would not have been able to write that Ethereum logic.  My only question is: I tried the test steps and get an `Error: Provided address "ADD YOUR ADDRESS HERE" is invalid, the capitalization checksum test failed, or its an indrect IBAN address which can't be converted.` , so I replaced "ADD YOUR ADDRESS HERE" with the commented address to the right (because I'm too lazy to look up my ethereum address).  And I still got an invalid address error: `Error: Provided address "1.189349833453414e+48" is invalid,` ...  Is that how it's supposed to work? 

I also get an error about resolving the PayID when I try the other example: 
Screen Shot 2020-04-23 at 12 44 39 PM

Am I doing things wrong?

**Comment by Stormtv** [comment]
@amiecorso For the ETH demo you need to insert an address and private key. I sent you some credentials over slack to test with.

**Review by amiecorso — APPROVED** [review]
ah gotcha, thanks!

File: node/src/index-send-eth-to-payid.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
+const axios = require('axios')
```
Comment (keefertaylor) [inline node/src/index-send-eth-to-payid.js:1]: Can we just drop this into pay-id-demo? 

If you look at the XRP demo we generally just shove a bunch of functionality after each other. I think this makes it easier to find examples by a single high level ordering.

File: node/src/index-send-eth-to-payid.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+const EthereumTx = require('ethereumjs-tx').Transaction
+const open = require('open')
+const Web3 = require('web3')
+const { PayIDClient } = require('xpring-js')
+
+/**
+ * The address you are sending from
+ * @type {String}
+ */
+const fromAddress = 'ADD YOUR ADDRESS HERE' // 0xD0544A81F0962F7ff0d08Eaf1E5B226DAD331316
```
Comment (keefertaylor) [inline node/src/index-send-eth-to-payid.js:11]: Any reason to not make this a test-net demo and to just provide these values up front?

File: node/src/index-send-eth-to-payid.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+/**
+ * The private key of the account you are sending from
```
Comment (keefertaylor) [inline node/src/index-send-eth-to-payid.js:14]: assume this is hex encoded? Can we notate that if we're not providing an example value?

File: node/src/index-send-xrp-to-payid.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
+const open = require('open')
```
Comment (keefertaylor) [inline node/src/index-send-xrp-to-payid.js:1]: This demo already exists in index-xpring.js

File: node/src/index-send-eth-to-payid.js (hunk 1 above)
Comment (Stormtv) [reply node/src/index-send-eth-to-payid.js:1]: I think the ETH stuff should be separated out for ease of finding

File: node/src/index-send-eth-to-payid.js (hunk 2 above)
Comment (Stormtv) [reply node/src/index-send-eth-to-payid.js]: Added working values

File: node/src/index-send-eth-to-payid.js (hunk 3 above)
Comment (Stormtv) [reply node/src/index-send-eth-to-payid.js:14]: Provided an example for this

File: node/src/index-send-xrp-to-payid.js (hunk 4 above)
Comment (Stormtv) [reply node/src/index-send-xrp-to-payid.js]: deleted this one since it is duplicative

File: node/src/index-send-eth-to-payid.js [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+const EthereumTx = require('ethereumjs-tx').Transaction
+const open = require('open')
+const Web3 = require('web3')
+const { PayIDClient } = require('xpring-js')
+
+/**
+ * The address you are sending from
+ * @type {String}
+ */
+const fromAddress = '0xffff84165744164EAd44C6DEA01dE6DcB4e85A9f' // 0xD0544A81F0962F7ff0d08Eaf1E5B226DAD331316
```
Comment (Stormtv) [inline node/src/index-send-eth-to-payid.js:11]:
```suggestion
const fromAddress = '0xffff84165744164EAd44C6DEA01dE6DcB4e85A9f' // INSERT YOUR ADDRESS HERE
```

**Review by keefertaylor — APPROVED** [review]

**Pull request merged**